IPython/terminal/ipapp.py

```diff
@@ -154,6 +154,7 @@ def make_report(self,traceback):
 
 aliases = dict(base_aliases)
 aliases.update(shell_aliases)
+aliases['term-title-format'] = 'TerminalInteractiveShell.term_title_format'
```

Comment:
Thanks ! I'm make the `--help-all`  [more visible](https://github.com/ipython/traitlets/pull/404)